caso/extract/manager.py

```diff
@@ -19,13 +19,12 @@
 from dateutil import tz
 from oslo_config import cfg
 from oslo_log import log
-from oslo_utils import importutils
 import six
+from stevedore import extension
+
+manager = extension.ExtensionManager(namespace='extractors',
```

Comment:
I will put the namespace in a global NAMESPACE variable, and use "caso.extractors" as its value.